packages/core/src/external/carequality/dr/get-document-download-url.ts

```diff
@@ -12,7 +12,7 @@ import { XDSRegistryError } from "../error";
 const region = Config.getAWSRegion();
 const medicalDocumentsBucketName = Config.getMedicalDocumentsBucketName();
 
-export async function getDocumentDownloadURL(
+export async function constructDocumentReferences(
```

Comment:
nit: construct > build

We use `build` throughout the code to indicate building something (and `make` for tests, usually). By reusing verbs/actions it makes it easier to find the function we're looking for.

Also, construct is also a noun, which makes `build` clearer.